bool on_ramp
float32 dist_to_ramp
float32 ang_lidar
float32 dist_on_ramp
float32 ang_imu
